### ISSUE TITLE
[Feature]: Add EstimateTradeBasedOnPriceImpact to osmosisd CLI

### ISSUE THREAD

**Issue by jasbanza** [osmosis-labs/osmosis#6711]
### Summary

Make EstimateTradeBasedOnPriceImpact (as seen in PR #6167) available as an osmosisd poolmanager query.

### Problem Definition

The recently added EstimateTradeBasedOnPriceImpact query should be made available on osmosisd CLI

### Proposed Feature

Add EstimateTradeBasedOnPriceImpact to osmosisd CLI

**Comment by DongLieu** [comment]
can i handle it?

**Comment by DongLieu** [comment]
I saw it[ here](https://github.com/osmosis-labs/osmosis/blob/main/x/poolmanager/client/cli/query.go#L34)

**Comment by jasbanza** [comment]
> I saw it[ here](https://github.com/osmosis-labs/osmosis/blob/main/x/poolmanager/client/cli/query.go#L34) 

Yes this is the old one though. It doesn't accept any argument for price impact.
